Tests/Unit/Document/AbstractGetterSetterTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\MagentoConnectorBundle\Tests\Unit\Document;
+
+use ONGR\MagentoConnectorBundle\Tests\app\fixtures\ExpectedDocuments\ExpectedDocuments;
+
+abstract class AbstractGetterSetterTest extends \PHPUnit_Framework_TestCase
```

Comment:
Does https://github.com/ongr-io/ConnectionsBundle/blob/8d1274010112fec7653cdaf78eef3d8bf3e9eb03/Tests/Unit/Entity/AbstractEntityTest.php do the same? Maybe register issue and leave as is currently?